docs/velox-backend-scalar-function-support.md

```diff
@@ -187,20 +187,20 @@
 
 ## Lambda Functions
 
-| Spark Functions   | Spark Expressions   | Status   | Restrictions   |
-|-------------------|---------------------|----------|----------------|
-| aggregate         | ArrayAggregate      | S        |                |
-| array_sort        | ArraySort           | S        |                |
-| exists            | ArrayExists         | S        |                |
-| filter            | ArrayFilter         | S        |                |
-| forall            | ArrayForAll         | S        |                |
-| map_filter        | MapFilter           |          |                |
-| map_zip_with      | MapZipWith          | S        |                |
-| reduce            | ArrayAggregate      | S        |                |
-| transform         | ArrayTransform      | S        |                |
-| transform_keys    | TransformKeys       | S        |                |
-| transform_values  | TransformValues     | S        |                |
-| zip_with          | ZipWith             | S        |                |
+| Spark Functions   | Spark Expressions   | Status | Restrictions   |
```

Comment:
This file is automatically generated and updated by script. Please avoid changing the format of this file.